appveyor.yml

```diff
@@ -8,19 +8,37 @@ init:
         Import-Module '..\appveyor-tool.ps1'
 
 install:
-  ps: Bootstrap
+  - ps: Bootstrap
+  - cinst pandoc
+  - ps: $env:Path += ";C:\Program Files\Pandoc\"
 
 cache:
   - C:\RLibrary
+  - C:\ProgramData\chocolatey\bin -> appveyor.yml
+  - C:\ProgramData\chocolatey\lib -> appveyor.yml
```

Comment:
what is `chocolatey` and what does the `-> appveyor.yml` do? (I'm not very familiar with powershell)